src/pyAEATsii/mapping.py

```diff
@@ -138,7 +138,7 @@ def build_issued_invoice(self, invoice):
             # TODO: Macrodato
             # TODO: DatosInmueble
             # TODO: ImporteTransmisionInmueblesSujetoAIVA
-            # TODO: EmitidaPorTercerosODestinatario
+            'EmitidaPorTercerosODestinatario': self.emited_by_party(invoice),
```

Comment:
I'd rather see something like
`'EmitidaPorTercerosODestinatario': 'S' if self.emited_by_party(invoice) else 'N',`
so that implementers of the mappers are free from the burden of the actual SII keys to be sent and can return a naive python boolean.

Therefore the default implementation should be `return False`

Reply:
Thinking thoroughly, we have been fooled by a false friend. The function should be named `issued_by_party`!